### PR TITLE
refactor/create home page

### PR DESCRIPTION
**Pull request by NazarSak**

### CODE REVIEW

**Comment by MaksVibe** [comment]
We are working from "client" directory. You've placed project at the root. Please place your changes in "client" directory for us to make the merge. Thanks

File: client/app/page.tsx
Comment (MaksVibe) [inline client/app/page.tsx:1]: remove, pls, unnecessary changes, it doesn't belong to your branch

File: client/bash.exe.stackdump
Comment (MaksVibe) [inline client/bash.exe.stackdump:1]: remove, pls, unnecessary changes, it doesn't belong to your branch

File: client/components/StartPopUp/StartPopUp.tsx [hunk 1]
```diff
@@ -1,27 +1,26 @@
 'use client';
 
 import React, { useState } from 'react';
-
 import Agreement from 'components/Agreement/Agreement';
 import Condition from 'components/Condition/Condition';
-import { Button,Wrapper } from 'components/Condition/Condition.styles';
+import { Wrapper, Button } from './StartPopUp.styles';
```
Comment (DStukalo) [inline client/components/StartPopUp/StartPopUp.tsx]: We have general btn component

File: client/components/StartPopUp/StartPopUp.tsx [hunk 2]
```diff
@@ -1,27 +1,26 @@
 'use client';
 
 import React, { useState } from 'react';
-
 import Agreement from 'components/Agreement/Agreement';
 import Condition from 'components/Condition/Condition';
```
Comment (DStukalo) [inline client/components/StartPopUp/StartPopUp.tsx]: If these components are used only here, then let's immediately put them in the folder with this component

**Review by MaksVibe — APPROVED** [review]
